.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -18,7 +18,8 @@ Learn more about contributing: [CONTRIBUTING.md](https://github.com/nf-core/deta
 - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/detaxizer/tree/master/.github/CONTRIBUTING.md)
 - [ ] If necessary, also make a PR on the nf-core/detaxizer _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
 - [ ] Make sure your code lints (`nf-core lint`).
-- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
+- [ ] Ensure the test suite passes (`nf-test test main.nf.test -profile test,docker`).
```

Comment:
When trying to run that command in the pipeline root dir:

```
$ nf-test test main.nf.test -profile test,docker
Unknown option: '-profile'
Possible solutions: --profile
```

Reply:
see the comment above

Reply:
Not sure which comment you are referring to.

Reply:
https://github.com/nf-core/detaxizer/pull/22#discussion_r1528399908